tests/serde/mod.rs

```diff
@@ -770,36 +772,42 @@ fn utc_offset_partial() {
         &[Token::Tuple { len: 3 }, Token::TupleEnd],
         "expected offset hours",
     );
-    assert_de_tokens_error::<Compact<UtcOffset>>(
+    assert_de_tokens_error::<Readable<UtcOffset>>(
+        &[Token::Tuple { len: 3 }, Token::TupleEnd],
+        "expected offset hours",
+    );
+
+    let value = UtcOffset::from_hms(23, 0, 0).unwrap();
```

Comment:
This (and other instances in the file) can be turned into a macro invocation.

Reply:
done

Reply:
No need to specify the zero values :)

Reply:
This is the only thing remaining, then I'll merge the PR :slightly_smiling_face:

Reply:
Finish

Reply:
No it's not? For example, `offset!(+23:0:0)` should be `offset!(+23)`.

Reply:
Ok I see, has been revised